src/Factory/FormFactory.php

```diff
@@ -20,10 +22,12 @@
 final class FormFactory
 {
     private FormFactoryInterface $symfonyFormFactory;
+    private AdminUrlGenerator $adminUrlGenerator;
 
-    public function __construct(FormFactoryInterface $symfonyFormFactory)
+    public function __construct(FormFactoryInterface $symfonyFormFactory, AdminUrlGenerator $adminUrlGenerator)
```

Comment:
```suggestion
    public function __construct(FormFactoryInterface $symfonyFormFactory, AdminUrlGeneratorInterface $adminUrlGenerator)
```

IIRC it is already a thing, isn't it?

Reply:
Yes! I've just updated this. Thanks.